modules/govuk_jenkins/templates/jobs/drug_email_alert_check.yaml.erb

```diff
@@ -20,6 +14,10 @@
         - timed: '21 * * * *' # every hour on the 21st minute
     builders:
         - shell: |
+            # If GitHub is down, re-use the directory we cloned from last time the job ran.
+            (git clone https://github.com/alphagov/email-alert-monitoring tmp && rm -rf email-alert-monitoring && mv tmp email-alert-monitoring) || true
```

Comment:
If this fails, can we mark the build as Unstable, but not alert through pagerduty? Same for Travel Advice, obvs.

Reply:
Just to clarify, is this what you mean:

If the clone succeeds and the email check succeeds
  - success
If the clone fails and the email check succeeds
  - unstable
If the clone succeeds and the email check fails
  - failure (+ trigger pagerduty)
If the clone fails and the email check fails
  - failure (+ trigger pagerduty)

Reply:
Yep 👍